### PR TITLE
Various updates

### LINKED ISSUE #48: Fix dynamic impact on V-72305 for TFPT server
We do not set impact on the negative case correctly

### PR DESCRIPTION
**Pull request by Bialogs**
This pull request closes #48 and fixes the `TODO` in `controls/V-72039.rb#56`. Beyond that:
* I've updated and expanded comments in the `attributes.yml` 
* Check for the RedHat release per the STIG for V-72241 and V-72251.

### CODE REVIEW

**Comment by Bialogs** [comment]
That sounds better; I will switch over to that.

**Comment by trevor-vaughan** [comment]
@Bialogs Could you check the Travis failures as well? I think this may be a new inspec thing popping up.

**Comment by Bialogs** [comment]
As it turns out the InSpec virtualization resource does not detect VMWare which is why `/dev/vmci` exists anyway so there's not really a point to using the InSpec virtualization resource.

**Comment by Bialogs** [comment]
Was something changed in SIMP recently that updated the SELinux context on `/dev/vmci` from `vmci_device_t` to `device_t`?

**Comment by Bialogs** [comment]
Force push to get travis to rebuild after timeout...

File: controls/V-72039.rb [hunk 1]
```diff
@@ -50,14 +56,21 @@
...
 # sudo rpm -Uvh <packagename>"
+
   tag "fix_id": "F-78391r1_fix"
   tag "dangerous": { :reason => "Uses global find command" }
 
-  # @todo - check for host vm device files as mentioned and pass if found
   findings = Set[]
   findings = findings + command('find / -context *:device_t:* \( -type c -o -type b \) -printf "%p %Z\n"').stdout.split("\n")
   findings = findings + command('find / -context *:unlabeled_t:* \( -type c -o -type b \) -printf "%p %Z\n"').stdout.split("\n")
+  findings = findings + command('find / -context *:vmci_device_t:* \( -type c -o -type b \) -printf "%p %Z\n"').stdout.split("\n")
```
Comment (aaronlippold) [inline controls/V-72039.rb]: aaronl avoids the temtation to use xargs -i here ... :)

File: controls/V-72241.rb [hunk 2]
```diff
@@ -59,7 +59,14 @@
 The SSH service must be restarted for changes to take effect."
   tag "fix_id": "F-78595r3_fix"
 
-  describe sshd_config do
-    its('ClientAliveCountMax') { should cmp '0' }
+  if os.release.to_f >= 7.4
+    impact 0.0
+    describe "The release is #{os.release}" do
+      skip "The release is newer than 7.4; this control is Not Applicable."
```
Comment (aaronlippold) [inline controls/V-72241.rb]: You need to update the impact logic as well so that this buckets to Not Applicable when NA.

Comment (Bialogs) [reply controls/V-72241.rb]: Impact is set to `0.0` the line above.

**Comment by trevor-vaughan** [comment]
> 
> 
> Was something changed in SIMP recently that updated the SELinux context on `/dev/vmci` from `vmci_device_t` to `device_t`?

Nope, that's not a SIMP thing. It may be a vendor thing though.

File: controls/V-72305.rb [hunk 3]
```diff
@@ -37,11 +41,13 @@
...
 
-  describe command('grep server_args /etc/xinetd.d/tftp') do
-    its('stdout.strip') { should match %r{^\s*server_args\s+=\s+-s\s\S+\s*$} }
-  end if package('tftp').installed?
-
-  describe "The TFTP package is not installed" do
-    skip "If a TFTP server is not installed, this is Not Applicable."
-  end if !package('tftp').installed?
+  if package('tftp-server').installed?
+    describe command('grep server_args /etc/xinetd.d/tftp') do
```
Comment (aaronlippold) [inline controls/V-72305.rb]: technically we should use the `file` resource here and say `its('content') { should match .... }`

**Pull request merged**